src/pages/Payment.vue

```diff
@@ -4,7 +4,7 @@
 
             <div class="content fullwidth">
                 <h1 v-if="vouchersAdded" v-on:click="collapsed = !collapsed" class="expandable" v-bind:class="{'expanded' : !collapsed}">You can request payment for <strong>{{ voucherCount }}</strong> voucher<span v-if="voucherCount > 1">s</span>.</h1>
-                <h1 v-else>You haven't added any vouchers yet.</h1>
+                <h1 v-else>[xXx] You don't have any pending vouchers at the moment.</h1>
```

Comment:
Your vouchers aren't pending payment at this stage, they will be *after* the person requests payment;
try "[xXx] There are no vouchers to request payment for. Add some!" or something.